Cargo.toml

```diff
@@ -31,8 +33,10 @@ harness = false
 name = "benches"
 
 [dependencies]
-log = "0.3.9"
-slog = "2.2"
+log = { version = "0.3.9", features = ["release_max_level_info"] }
```

Comment:
why do we still use 0.3.9?

Reply:
I've tried to use 0.4.x, but file names will become `<unknown>`. Anyway, under the hood, there will be always 0.4.x and 0.3.x because:
- slog-stdlog depends on 0.3.x
- we use `set_max_level ` interface from 0.4.x (which is not available in 0.3.x)

Reply:
oh, horrible...

can we fix the Unknown problem of 0.4?

Reply:
We will need our own fork of slog-stdlog to fix it, since the owner repo seems to be not maintained for a while (actually there is already a PR to upgrade dependency but no one took a look).

Reply:
I prefer fork because we can use `replace` in Cargo :-)

Putting two version of same crates seems too weird...